WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -159,18 +159,18 @@ class AccountSettingsService {
         let request = NSFetchRequest(entityName: ManagedAccountSettings.entityName)
         request.predicate = NSPredicate(format: "account.userID = %d", userID)
         request.fetchLimit = 1
-        let results = (try? context.executeFetchRequest(request) as! [ManagedAccountSettings]) ?? []
+        let results = (try? context.executeFetchRequest(request)) as? [ManagedAccountSettings] ?? []
```

Comment:
Nice job on this change.  However I feel this code is suffering in readability due to too much stuff going on in a single line.

Can we go for something like this instead? (this is untested, but just to convey the idea)

```
guard let results = (try? context.executeFetchRequest(request)) as? [ManagedAccountSettings] else {
    return nil
}

return results.first
```